src/resources/plsPlusAddress/script/defaultValue.js

```diff
@@ -0,0 +1,61 @@
+const bodyElem = document.querySelector('body');
+const plsPlusInit = bodyElem.getAttribute('data-plsPlusInit');
+
+function getFranchise(myFranchise) {
+  if (myFranchise === "auto") {
+    const host = location.host;
+    if (host.indexOf("forgov") !== -1) {
+      return "forgov";
+    } else if (host.indexOf("tmr") !== -1) {
+      return "tmr";
+    } else if (host.indexOf("covid19") !== -1) {
+      return "covid19";
+    } else if (host.indexOf("health") !== -1) {
+      return "health";
+    // https://www.resource.qld.gov.au/ is not found
```

Comment:
https://www.resources.qld.gov.au/ does exist.

Reply:
vani has suggested me to remove automatically assigning franchise key, and the user will be forced to provide the key everytime they create an instance.
so i'll have to remove this function, what do you think?

Reply:
google maps key is franchise choice, you already allow the key to be replaced/overridden. I would be more inclined to put out a console.log message if franchise can't be chosen and key override is not injected. (could be include on the drop down when out of spec)

Reply:
i think vani's intention is to reduce the maintainence of this piece of code, to avoid somthing like if the mapping is different then we'll have to update every instances that have been used in different forms, do you think it's justify to remove this function?

Reply:
can you look up project properties? set once and forgot instead of per form?

Reply:
yeah that's in one solution i was thinking before, this function can be set in project's `custom JS>form module`
but in this case there is one extra step for them to remember to copy the script if they set up a new project...

Reply:
this should be added to the https://github.com/qld-gov-au/formio-helper-modules/tree/master/projectTemplates system which add's the default resources etc

Reply:
great, i'll try that out